src/io/reader.cpp

```diff
@@ -11,71 +11,109 @@ namespace copc
 void Reader::InitReader()
 {
 
-    if (!this->in_stream_->good())
+    if (!in_stream_->good())
         throw std::runtime_error("Invalid input stream!");
 
-    this->reader_ = std::make_unique<lazperf::reader::generic_file>(*this->in_stream_);
+    reader_ = std::make_unique<lazperf::reader::generic_file>(*in_stream_);
 
-    auto header = las::LasHeader::FromLazPerf(this->reader_->header());
+    auto header = las::LasHeader::FromLazPerf(reader_->header());
+
+    // Load vlrs and evlrs
+    auto vlrs = ReadVlrs();
```

Comment:
Are VlrHeader and EvlrHeader convertible? If not could we make our own VlrHeader class, so that we can support the use case of people storing projections or something in Evlrs? I.e. when looking for a certain vlr we can just iterate over one Vlrs list instead of Vlrs and Evlrs